src/NuGetCommon.props

```diff
@@ -21,9 +21,14 @@
 
     </PropertyGroup>
 
-    <XmlPoke XmlInputPath="Sdk/Sdk.props" 
-             Query="//ItemGroup//PackageReference[@Include='DSharp.Mscorlib']/@Version" 
+    <ItemGroup>
+      <VersionSyncedImports Include="DSharp.Mscorlib"/>
```

Comment:
What does this do?

Reply:
updates the version numbers for package references listed by the VersionSyncedImports ItemGroup, previously the reference to DSharp.Mscorlib was hardcoded in the poke task